pkg/monitortests/network/legacynetworkmonitortests/networking.go

```diff
@@ -21,6 +23,32 @@ type testCategorizer struct {
 	substring string
 }
 
+func getPlatformType(clientConfig *rest.Config) (configv1.PlatformType, error) {
+	var platform configv1.PlatformType
+
+	kubeClient, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return platform, fmt.Errorf("error creating kubeClient: %v", err)
+	}
+	isMicroShift, err := exutil.IsMicroShiftCluster(kubeClient)
+	if err != nil {
+		return platform, fmt.Errorf("error checking MicroShift cluster: %v", err)
+	}
+	if isMicroShift {
+		return platform, nil
```

Comment:
this will be empty, not nil, correct?  You sure that's what you want?

Reply:
Yes, the platform is only needed when deploying on Azure, and thats not the case for MicroShift.